src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -142,13 +167,13 @@ protected override void Append(StringBuilder builder, LogEventInfo logEvent)
                 var sb2 = new StringBuilder(128);
                 string separator = string.Empty;
 
-                foreach (ExceptionDataTarget targetRenderFunc in this.exceptionDataTargets)
+                foreach (ExceptionRenderingFormat renderingFormat in this.Formats)
                 {
                     sb2.Append(separator);
-                    targetRenderFunc(sb2, logEvent.Exception);
+                    _functions[renderingFormat](sb2, logEvent.Exception);
```

Comment:
please create a variable of `  _functions[renderingFormat]` (just for readability).